### PR TITLE
VCR: Add stats on credentials in store

### PR DESCRIPTION
**Pull request by reinkrul**
Closes https://github.com/nuts-foundation/nuts-node/issues/1432

E.g.:

```yaml
vcr:
    credential_count: 7
    issuer:
        issued_credentials_count: 0
        revoked_credentials_count: 0
    verifier:
        revocations_count: 18
```

### CODE REVIEW

File: vcr/vcr.go [hunk 1]
```diff
@@ -404,3 +404,29 @@ func (c *vcr) Untrusted(credentialType ssi.URI) ([]ssi.URI, error) {
...
+			Title: "issuer",
+			Items: c.issuerStore.Diagnostics(),
+		},
+		core.DiagnosticResultMap{
+			Title: "verifier",
+			Items: c.verifierStore.Diagnostics(),
+		},
+		core.GenericDiagnosticResult{
+			Title:   "credential_count",
+			Outcome: credentialCount,
```
Comment (gerardsn) [inline vcr/vcr.go:429]: What exactly are these?

Is the last one the 'holder'?

**Review by gerardsn — COMMENTED** [review]
I don't think I know what all the fields are based on these names. 
Can they be split based on 'my/node credentials' and 'network/all credentials'?

Should any of these also be prometheus metrics?

File: vcr/vcr.go (hunk 1 above)
Comment (reinkrul) [reply vcr/vcr.go:429]: @woutslakhorst ?

**Comment by woutslakhorst** [comment]
Descriptions can be added to docs.

File: vcr/vcr.go (hunk 1 above)
Comment (woutslakhorst) [reply vcr/vcr.go:429]: `credential_count` is global + holder
`verifier` holds globally revoked

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
Some info in the docs: monitoring and maybe backup/restore?

**Comment by woutslakhorst** [comment]
metrics are needed when you want to turn the number into a graph... Can't think of theause-case for that.

**Comment by reinkrul** [comment]
> metrics are needed when you want to turn the number into a graph... Can't think of a use-case for that.

Not required for metrics I think

File: docs/pages/deployment/monitoring.rst [hunk 2]
```diff
@@ -54,6 +61,13 @@ Returns the status of the various services in ``yaml`` format:
 
 If you supply ``application/json`` for the ``Accept`` HTTP header it will return the diagnostics in JSON format.
 
+Explanation of ambiguous/complex entries in the diagnostics:
```
Comment (gerardsn) [inline docs/pages/deployment/monitoring.rst:64]: Should value of -1 be explained?

**Review by gerardsn — APPROVED** [review]
I'd prefer some more self-explanatory naming like: (if correct of course)
```
    vcr:
         credential_count: 7
         revocation_count: 18
         issued:
             credential_count: 0
             revocation_count: 0
```
but that doesn't change the content.

File: docs/pages/deployment/monitoring.rst (hunk 2 above)
Comment (reinkrul) [reply docs/pages/deployment/monitoring.rst:64]: Not really I think, there will be an error in the log

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**